src/main/java/org/terasology/module/lightandshadow/LASUtils.java

```diff
@@ -162,6 +166,9 @@ public static String getOppositionTeam(String team) {
         if (team.equals(BLACK_TEAM)) {
             return RED_TEAM;
         }
+        if (team.equals(WHITE_TEAM)) {
+            return WHITE_TEAM;
+        }
```

Comment:
What do we need this for?

Reply:
this is just to prevent NPE that can occur on https://github.com/Terasology/LightAndShadow/blob/develop/src/main/java/org/terasology/module/lightandshadow/systems/PlayerDeathSystem.java#L71

Reply:
also this will prevent the player from facing to the direction of red/black team and will make the player to face magic fool